### PR TITLE
Implemented Ship and Module constructors

### PR DESCRIPTION
**Pull request by felixlinker**

### CODE REVIEW

File: src/Ship.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 
+import Ajv from 'ajv';
+import { cloneDeep, map } from 'lodash';
+
+export const VALIDATOR = new Ajv();
```
Comment (felixlinker) [inline src/Ship.js]: Should this be moved into a new module/file?

File: src/Ship.js [hunk 2]
```diff
@@ -293,3 +316,9 @@ class Ship {
  * @return {string}
  */
 export function compressJson(json) {}
+
+/**
+ * @param {string} compressedBuild
+ * @return {object}
+ */
+export function decompress(compressedBuild) {}
```
Comment (felixlinker) [inline src/Ship.js]: Same as with `VALIDATOR`?

File: src/Ship.js (hunk 1 above)
Comment (willyb321) [reply src/Ship.js]: Probably.

File: src/Ship.js (hunk 2 above)
Comment (willyb321) [reply src/Ship.js]: Yeah, probably into a util file.

**Pull request merged**